x-pack/plugins/enterprise_search/public/applications/shared/types.ts

```diff
@@ -27,8 +29,16 @@ export interface SchemaConflicts {
   [key: string]: SchemaConflictFieldTypes;
 }
 
+
 export interface IIndexingStatus {
   percentageComplete: number;
   numDocumentsWithErrors: number;
   activeReindexJobId: number;
 }
+
+export interface IndexJob extends IIndexingStatus {
+  isActive?: boolean;
+  hasErrors?: boolean;
+}
+
+export type TOperation = typeof ADD | typeof UPDATE;
```

Comment:
Should we remove `T` in the type name?
If yes, feel free to merge and fix in later PRs.

Reply:
Thanks. Will address this in a follow-up PR.